benchmarks/domains/synthetic_2C1D_1C.py

```diff
@@ -0,0 +1,115 @@
+"""Synthetic function with two continuous and one discrete input."""
+
+from uuid import UUID
+
+import numpy as np
+from numpy import pi, sin, sqrt
+from pandas import DataFrame
+
+from baybe.campaign import Campaign
+from baybe.parameters import NumericalContinuousParameter, NumericalDiscreteParameter
+from baybe.recommenders.pure.nonpredictive.sampling import RandomRecommender
+from baybe.searchspace import SearchSpace
+from baybe.simulation import simulate_scenarios
+from baybe.targets import NumericalTarget, TargetMode
+from benchmarks.definition import Benchmark
+from benchmarks.definition.config import (
+    DEFAULT_RECOMMENDER,
+    RecommenderConvergenceAnalysis,
+)
+
+
+def lookup(z: np.ndarray, x: np.ndarray, y: np.ndarray) -> np.ndarray:
+    """Lookup function.
+
+    Inputs:
+        z   discrete   {1,2,3,4}
+        x   continuous [-2*pi, 2*pi]
+        y   continuous [-2*pi, 2*pi]
+    Output: continuous
+    Best value: 4.09685
```

Comment:
since best value is relative to the optimization goal (minimization, amximization etc) the implied objective needs to be mentioned somewhere

Reply:
agree (ideally, we should have this later as class attribute when generalizing the domain definitions). For now, I can add it to the docstring, together with the optimal input value .... once I've figured that one out :D

Reply:
Do we even need this information? This is only something relevant for synthetic benchmarks, so we could either just remove this or replace by something like `(lower_bound, upper_bound)` which can then later be displayed. This avoids to have a super strict values as we could e.g. just have `4.1` here

Reply:
so for plots this is usually very useful to display the best possible value asa flat line
Having said that, it probably needs to be available programatically and not just as comment